inji-web/package.json

```diff
@@ -17,7 +15,6 @@
     "autoprefixer": "^9.8.6",
     "crypto-js": "^4.2.0",
     "i18next": "^23.11.2",
-    "inji-web": "file:",
```

Comment:
please remove this in both package and package-lock

Reply:
I had removed it already, it is not showing on my local system. will see what else can be done..

Reply:
@kamlesh012 , replace the package.json and package-lock.json from develop and commit it